packages/cx/src/widgets/grid/Grid.d.ts

```diff
@@ -73,6 +73,7 @@ interface GridGroupingConfig {
    caption?: StringProp;
    name?: StringProp;
    text?: StringProp;
+   comparer?: (a: any, b: any) => number;
```

Comment:
Typing for the comparer arguments could be improved. Instead of `any` we should introduce a type.
```ts
interface GroupingResult {
  key: any,
  aggregates: any,
  name: string,
  results: any[]
}
```
and whatever else is inside that object.

Reply:
The `results` field could also be typed similar to this:

```ts
interface GridProps<T = unknown>
```

Reply:
Sure, will make the change. It was a copy/paste from column comparer definition. 🙂

Reply:
#740